### PR TITLE
Update headshots

### PR DESCRIPTION
**Pull request by jacebrowning**

### CODE REVIEW

**Review by Ross-Hunter — APPROVED** [review]
🍰

File: readme.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+```
+
+# Development
+
+Run the development server:
+
+```
+$ hugo serve
+```
+
```
Comment (Ross-Hunter) [inline readme.md:24]: Deploy with `./deploy.sh`

Comment (Ross-Hunter) [reply readme.md:24]: I should probably make this a private repo...

**Comment by Ross-Hunter** [comment]
This needs to actually be deployed.

**Comment by jacebrowning** [comment]
I ran the script and it pushed to `gh-pages`, but I don't see the live changes. Are we caching?

**Comment by Ross-Hunter** [comment]
Yep, the cache takes a bit to expire, even when manually purging it.

When you run the deploy script you don't need to manually push the branch, the deploy script should handle that for us.

The deploy script is just what came with the template, I'm open to other options.

**Pull request merged**